### DISCUSSION TITLE
Just to say thank you.

### DISCUSSION THREAD

**Discussion by renardsas** [okikio/spring-easing#5] [General]
This library is really great.  🙏 🙏 🙏
Well documented, both in the code and website.

Just a small note, the documentation is technically perfect, but maybe another version that is more "beautifully presented" using a tool like [Docus ](https://docus.dev/) could be helpful for beginners.

**Comment by okikio** [comment]
Thanks you, I'm always working on improving the docs, so I'll take a look at Docus 👍
